rewrite/src/tests/test.c

```diff
@@ -50,12 +49,13 @@ void LoadTestPatches()
     FlushCache();
 }
 
-u32 PatchFunction_Beg(u32* address)
+u32 PatchFunction_Beg(u32* address, const char* funcName)
 {
     u32 addr = (u32)address;
+    s_nameTestedFunc = funcName;
     __asm__ volatile("move $k1, %0" : : "r"(addr));
 
-    u32 index = 0;
+    u32 index = UINT32_MAX;
```

Comment:
to match our integral types paradigm, should this be named U32_MAX instead?

Reply:
this is a stdint macro, i guess i could define our macros to match yea